target/scripts/update-check.sh

```diff
@@ -41,20 +38,19 @@ There is a docker-mailserver update available on your host: $(hostname -f)
 Current version: ${VERSION}
 Latest  version: ${LATEST}
 
-Changelog: ${CHANGELOG}
+Changelog: ${CHANGELOG_URL}
 EOM
       echo "${MAIL}" | mail -s "Mailserver update available! [ ${VERSION} --> ${LATEST} ]" "${POSTMASTER_ADDRESS}" && \
 
-      _log "Info: Update available [ ${VERSION} --> ${LATEST} ]" && \
-
       # only notify once
-      exit 0
+      _log_with_date 'info' "Update available [ ${VERSION} --> ${LATEST} ]" && exit 0
```

Comment:
I find the old structure more clear. "only notify once" explains `exit 0`

Reply:
With all due respect, I had difficulties seeing that an

``` BASH
... && \

# comment here
exit 1
```

all belongs together, because I had never seen a line break escape following a comment following the next command to execute if the command on the first line succeeded. Therefore, I disagree, I think the new structure is way more pleasant to read.

Reply:
```bash
echo "${MAIL}" | mail -s "Mailserver update available! [ ${VERSION} --> ${LATEST} ]" "${POSTMASTER_ADDRESS}" && \
_log_with_date 'info' "Update available [ ${VERSION} --> ${LATEST} ]" && \
# only notify once
exit 0
```

If you still want the `exit` on the same line as `_log_with_date`, then at least the empty line between `echo` and `_log_with_date` should be removed, to make clear `_log_with_date` depends on the success of sending the mail.

Reply:
Or even better, remove `&& \` from the `echo` line. The info should be logged, regardless of the success of sending the mail.

Reply:
On the other, then there will be no further tries to send the email, because `exit 0` will stop the update-check.

Rare case: `update-check` tries to send the email, while `check-for-changes` kicks in and restarts postfix at the same time. Mail deliver fails, "update available" is logged and `update-check stops`. The admin won't be notified by mail in that case.
But this is the primary aim of the `update-checker`.

So I propose to either remove the empty line or take my proposal above with no empty line and exit on a separate line.

Reply:
Final final proposal 😆

I switched the `_log_with_date` and `echo` line. This means

1. Create a log entry every time
2. Stop `update-check` only, if mail was successful sent.

```bash
_log_with_date 'info' "Update available [ ${VERSION} --> ${LATEST} ]"
echo "${MAIL}" | mail -s "Mailserver update available! [ ${VERSION} --> ${LATEST} ]" "${POSTMASTER_ADDRESS}" && \
# only notify
exit 0
```

or

```bash
_log_with_date 'info' "Update available [ ${VERSION} --> ${LATEST} ]"
# only notify
echo "${MAIL}" | mail -s "Mailserver update available! [ ${VERSION} --> ${LATEST} ]" "${POSTMASTER_ADDRESS}" && exit 0
```

Reply:
Done in e2211394774b18cf2906c4e59e4cd7278562077e